test/test_playbooks/fixtures/organization-0.yml

```diff
@@ -1,255 +1,262 @@
+    uri: https://or.deploy7.dev.atix/api/status
   response:
-    body:
-      string: !!binary |
-        H4sIAAAAAAAAA43OPQqAMAwF4N1TSGYXxUG9TAlasdg/mtRFendtN6c6hTy+8NK2NxAjR4IF3Akd
-        kAyXWuW73+Cj9nl+xRYDsnJWmBwNE6SuSIGRjxrvx8xXtJuWXtkanz/6V0M52V2QBq1gpJOqP0FK
-        qXkAlEOknQsBAAA=
+    body: {string: !!python/unicode '{"result":"ok","status":200,"version":"1.21.3","api_version":2}'}
     headers:
-      apipie-checksum: [96d85e8c4306b45b12132dcec2243d18d47f3d81]
+      apipie-checksum: [1226a706415b816a8d8e0606e65431f4e2ceadd3]
       cache-control: ['max-age=0, private, must-revalidate']
-      connection: [Keep-Alive]
-      content-encoding: [gzip]
-      content-length: ['125']
+      content-length: ['63']
       content-security-policy: ['default-src ''self''; child-src ''self''; connect-src
           ''self'' ws: wss:; img-src ''self'' data: *.gravatar.com; script-src ''unsafe-eval''
           ''unsafe-inline'' ''self''; style-src ''unsafe-inline'' ''self''']
       content-type: [application/json; charset=utf-8]
-      date: ['Tue, 16 Oct 2018 07:37:13 GMT']
-      etag: [W/"6c4ecb1747e00c916f1b93bd5b9a6cd5-gzip"]
+      date: ['Wed, 22 May 2019 10:30:57 GMT']
+      etag: [W/"8ae0d3efe0e232e2a73486dc1f2e0fbb"]
       foreman_api_version: ['2']
-      foreman_version: [1.19.0]
-      keep-alive: ['timeout=5, max=10000']
-      server: [Apache/2.4.6 (CentOS)]
-      set-cookie: [_session_id=201399c8a05414eb7823bed442fe02d8; path=/; secure; HttpOnly;
+      foreman_version: [1.21.3]
+      server: [Apache]
+      set-cookie: [_session_id=54e1b7a2951d29868c22f3aefeeebece; path=/; secure; HttpOnly;
           SameSite=Lax]
       status: [200 OK]
       strict-transport-security: [max-age=631139040; includeSubdomains]
-      vary: [Accept-Encoding]
       x-content-type-options: [nosniff]
       x-download-options: [noopen]
       x-frame-options: [sameorigin]
       x-permitted-cross-domain-policies: [none]
-      x-powered-by: [Phusion Passenger 4.0.53]
-      x-request-id: [9eb483f2-ca11-4165-a67e-d606d449553d]
-      x-runtime: ['0.109987']
+      x-powered-by: [Phusion Passenger 4.0.18]
+      x-request-id: [0093ffaf-ac0b-492a-bd60-fc16464f3115]
+      x-runtime: ['0.099636']
       x-xss-protection: [1; mode=block]
     status: {code: 200, message: OK}
 - request:
-    body: !!python/unicode '{"search": "name=\"Test Organization\""}'
+    body: null
     headers:
-      Accept: ['*/*']
-      Accept-Encoding: ['gzip, deflate']
-      Connection: [keep-alive]
-      Content-Length: ['40']
-      User-Agent: [python-requests/2.19.1]
-      content-type: [application/json]
+      Accept: [application/json;version=2]
+      Cookie: [_session_id=54e1b7a2951d29868c22f3aefeeebece]
     method: GET
-    uri: https://foreman.example.com/katello/api/v2/organizations
+    uri: https://or.deploy7.dev.atix/katello/api/organizations?per_page=4294967296&search=name%3D%22Test+Organization%22&thin=False
```

Comment:
@sean797  it uses the katello endpoint. I do not know how it decides that.

Reply:
This depends on the apidoc. On Katello, you get

```
                        "apis": [
                            {
                                "api_url": "/katello/api/organizations",
                                "deprecated": null,
                                "http_method": "GET",
                                "short_description": "List all organizations"
                            }
                        ],
```

On Foreman, `/api/organizations`.

Given it never worked with plain Foreman before this PR…